config/config_test.go

```diff
@@ -221,8 +220,6 @@ deprecate-integer-display-length = true
 enable-enum-length-limit = false
 stores-refresh-interval = 30
 enable-forwarding = true
-[performance]
-txn-total-size-limit=2000
```

Comment:
There's a bug here. You can't remove `[performance]` because it causes `tcp-no-delay` to be a top-level config option, which is making this unit test fail:

```
morgo@ubuntu:~/go/src/github.com/morgo/tidb$ pkg=config make testpkg
Running unit test for github.com/pingcap/tidb/config
--- FAIL: TestConfig (0.00s)
    config_test.go:261: 
        	Error Trace:	config_test.go:261
        	Error:      	Received unexpected error:
        	            	config file config.toml contained invalid configuration options: tcp-no-delay; check TiDB manual to make sure this option has not been deprecated and removed from your TiDB version if the option does not appear to be a typo
        	Test:       	TestConfig
FAIL
coverage: 64.3% of statements
FAIL	github.com/pingcap/tidb/config	0.015s

```